.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules
 /dist
 /build
 /lib
-/.tmp
\ No newline at end of file
+/.tmp
+/.idea
```

Comment:
This should be in your global git ignore. It's specific to your setup and not this project.